report/finding.go

```diff
@@ -68,3 +98,29 @@ func maskSecret(secret string, percent uint) string {
 
 	return secret[:lth] + "..."
 }
+
+func (f *Finding) PrintAuxiliaryFindings() {
```

Comment:
Is this here instead of util because `auxiliaryFindings` is private? Would it make sense for the all the `Print*` functions to go in report since it seems like they're more focused on reporting info back to the user instead of the detection itself?